list/list.css

```diff
@@ -0,0 +1,78 @@
+@import url(/style.css);
+@import url(/avatar/avatar.css);
+.list-container {
+  margin: 2rem 0;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+.normal-list {
+  list-style-type: none;
+  border: 1px solid var(--dark-black);
+  padding: 0;
+  align-items: center;
+  background-color: var(--off-white);
+}
+.normal-list {
+  width: 50%;
+  min-width: 20rem;
+}
+.normal-list-item {
+  border-bottom: 1px solid var(--dark-black);
+  padding: 1rem;
+}
+.normal-list-item:nth-last-child(1),
+.stacked-list-item:nth-last-child(1) {
+  border-bottom: none;
+}
+.normal-list-item:hover,
+.stacked-list-item:hover {
+  background-color: var(--light-grey);
+}
+/* stacked list */
+.stacked-list {
+  padding: 0;
+  width: 50%;
+  min-width: 20rem;
+  border: 1px solid var(--dark-black);
+  background-color: var(--off-white);
+}
+.stacked-list-item {
+  padding: 1rem;
+  border-bottom: 1px solid var(--dark-black);
+  list-style-type: none;
+  display: flex;
```

Comment:
use utility classes for `flex`

Reply:
Sure, I'm adding it at last once the components are ready, so as to avoid confusion right now.